pipelines/cloud-platform-live-0/main/plan-environments.yaml

```diff
@@ -0,0 +1,66 @@
+resource_types:
+- name: pull-request
+  type: docker-image
+  source:
+    repository: teliaoss/github-pr-resource
+    #repository: jtarchie/pr
+    # repository: drnic/pullrequest-resource
+
+resources:
+- name: cloud-platform-environments-repo-pull-requests
+  type: pull-request
+  check_every: 1m
+  source:
+    repository: ministryofjustice/cloud-platform-environments
+    access_token: ((cloud-platform-environments-pr-git-access-token))
+- name: tools-image
+  type: docker-image
+  source:
+    repository: 926803513772.dkr.ecr.eu-west-1.amazonaws.com/cloud-platform/tools
+    tag: latest
+    aws_access_key_id: ((aws.access-key-id))
+    aws_secret_access_key: ((aws.secret-access-key))
+
+jobs:
+  - name: plan
+    serial: true
+    plan:
+      - get: cloud-platform-environments-repo-pull-requests
+        trigger: true
+        version: every
+      - put: cloud-platform-environments-repo-pull-requests
+        params:
+          path: cloud-platform-environments-repo-pull-requests
+          status: pending
+      - get: tools-image
+      - task: plan-environments
+        image: tools-image
+        config:
+          platform: linux
+          inputs:
+            - name: cloud-platform-environments-repo-pull-requests
+          params:
+            AWS_ACCESS_KEY_ID: ((aws.access-key-id))
+            AWS_SECRET_ACCESS_KEY: ((aws.secret-access-key))
+            KUBECONFIG: /tmp/kubeconfig
+            TF_PLUGIN_CACHE_DIR: /tmp/terraform-plugin-cache"
+          run:
+            path: /bin/sh
+            dir: cloud-platform-environments-repo-pull-requests
+            args:
+              - -c
+              - |
+                mkdir -p "${TF_PLUGIN_CACHE_DIR}"
+                aws s3 cp s3://cloud-platform-concourse-build-environments/kubeconfig /tmp/kubeconfig
+                ./bin/plan
+        on_failure:
+            put: cloud-platform-environments-repo-pull-requests
+            params:
+              path: cloud-platform-environments-repo-pull-requests
+              status: failure
```

Comment:
Looking at the original issue https://github.com/ministryofjustice/cloud-platform/issues/435, looks like we can possibly do the comment bit as well? Using a combination of `tee` and `grep` to extract the line and the resource's `comment_file` option to set it. Does that sound doable?

Reply:
comment_file is not working as expected, so not added that changes in.